docs/standard/choosing-core-framework-server.md

```diff
@@ -1,108 +1,106 @@
 
 * You have cross-platform needs.
 * You are targeting microservices.
 * You are using Docker containers.
-* You need high performance and scalable systems.
-* You need side by side of .NET versions by application.
+* You need high-performance and scalable systems.
+* You need side-by-side .NET versions per application.
 
-You should use .NET Framework for your server application when:
+Use .NET Framework for your server application when:
 
-* Your application currently uses .NET Framework (recommendation is to extend instead of migrating)
-* You need to use third-party .NET libraries or NuGet packages not available for .NET Core.
-* You need to use .NET technologies that are not available for .NET Core.
-* You need to use a platform that doesn’t support .NET Core.
+* Your app currently uses .NET Framework (recommendation is to extend instead of migrating).
+* Your app uses third-party .NET libraries or NuGet packages not available for .NET Core.
+* Your app uses .NET technologies that aren't available for .NET Core.
+* Your app uses a platform that doesn’t support .NET Core.
 
 ## When to choose .NET Core
 
-The following is a more detailed explanation of the previously-stated reasons for picking .NET Core.
+The following sections give a more detailed explanation of the previously stated reasons for picking .NET Core.
 
 ### Cross-platform needs
 
-Clearly, if your goal is to have an application (web/service) that should be able to run across platforms (Windows, Linux and macOS), the best choice is to use .NET Core.
+If your goal is to have an application (web/service) that should be able to run across platforms (Windows, Linux, and macOS), the best choice is to use .NET Core.
 
-.NET Core also supports the previously-mentioned operating systems as your development workstation. Visual Studio provides an Integrated Development Environment (IDE) for Windows.  You can also use Visual Studio Code on macOS, Linux and Windows which fully support .NET Core, including IntelliSense and debugging. You can also target .NET Core with most third-party editors like Sublime, Emacs, VI and can get editor IntelliSense using the open source [Omnisharp](http://www.omnisharp.net/) project. You can also avoid any code editor and directly use the .NET Core command-line tools, available for all supported platforms.
+.NET Core also supports the previously mentioned operating systems as your development workstation. Visual Studio provides an Integrated Development Environment (IDE) for Windows.  You can also use Visual Studio Code on macOS, Linux, and Windows, which fully supports .NET Core, including IntelliSense and debugging. You can also target .NET Core with most third-party editors like Sublime, Emacs, VI and can get editor IntelliSense using the open source [Omnisharp](http://www.omnisharp.net/) project. You can also avoid any code editor and directly use the .NET Core command-line tools, available for all supported platforms.
```

Comment:
.NET Core ~also~ supports development on Window, macOS, and Linux. Visual Studio provides an Integrated Development Environment (IDE) for Windows.  Visual Studio Code (VS Code) runs on macOS, Linux, and Windows. VS Code supports .NET Core, including IntelliSense and debugging. Most third-party editors (for example, Sublime, Emacs, and VI) work with .NET Core. These third-party editors get editor IntelliSense using the open source [Omnisharp](http://www.omnisharp.net/) project. 

Can you just say Omnisharp? These third-party editors get editor IntelliSense using [Omnisharp](http://www.omnisharp.net/).